gm4_vecto_shamir/data/gm4_vecto_shamir/functions/vehicle/spawn_minecart.mcfunction

```diff
@@ -0,0 +1,14 @@
+#run from active
+#@s = player with shamir equipped, sneaking on rails
+
+tag @s add gm4_ml_invehicle
+
+# spawn minecart
+summon minecraft:minecart ^ ^.1 ^ {Invulnerable:1b,Tags:["gm4_ml_vehicle"]}
+
+# link player to minecart with id
+execute store result score @s gm4_ml_vecto_id run data get entity @s UUID[0]
+scoreboard players operation @e[type=minecart,tag=gm4_ml_vehicle,distance=..1,sort=nearest,limit=1] gm4_ml_vecto_id = @s gm4_ml_vecto_id
+
+# start loop for check not in vehicle
+schedule function gm4_vecto_shamir:vehicle/check 40t
```

Comment:
In theory this should give players 2 seconds to get into the vehicle, but in a multiplayer setting, this will not be the case. temp vehicles should get a timer score that goes counts to 3, and once it's at 3 it will start checking if there's a player inside it.

Reply:
I know it resets the schedule timer when it's run again, but I've thought about it, and the alternative seems to be 2 more commands in main for the counter and checks. I'm not sure if that's worth it, since this "issue" would only occur if 2 or more players use this feature within 2 seconds of eachother, with the first player not actually getting in their vehicle, and if this does happen, it only means the vehicle is destroyed 2 seconds later than technically intended.